cli/src/quarantine_command.rs

```diff
@@ -0,0 +1,24 @@
+use clap::Args;
+
+use crate::upload_command::{run_upload, UploadArgs, UploadRunResult};
+
+#[derive(Args, Clone, Debug)]
+pub struct QuarantineArgs {
+    #[command(flatten)]
+    upload_args: UploadArgs,
+}
+
+pub async fn run_quarantine(QuarantineArgs { upload_args }: QuarantineArgs) -> anyhow::Result<i32> {
```

Comment:
optional: add comment that this is basically an alias. I think as `run_quarantine` matures, `run_upload` might start to lose meaning as a "run_upload_and_maybe_quarantine" function